src/webui/www/private/views/rssDownloader.html

```diff
@@ -0,0 +1,722 @@
+                width: 350,
+                height: 100
+            });
+        };
+
+        const renameRule = function(rule) {
+            new MochaUI.Window({
+                id: 'renameRulePage',
+                title: 'QBT_TR(Rule renaming)QBT_TR[CONTEXT=AutomatedRssDownloader]',
+                loadMethod: 'iframe',
+                contentURL: 'rename_rule.html?rule=' + encodeURIComponent(rule),
+                scrollbars: false,
+                resizable: false,
+                maximizable: false,
+                width: 350,
+                height: 100
+            });
+        };
+
+        const removeRule = function(rules) {
+            new MochaUI.Window({
+                id: 'removeRulePage',
+                title: 'QBT_TR(Rule deletion confirmation)QBT_TR[CONTEXT=AutomatedRssDownloader]',
+                loadMethod: 'iframe',
+                contentURL: 'confirmruledeletion.html?rules=' + encodeURIComponent(rules.join('|')),
+                scrollbars: false,
+                resizable: false,
+                maximizable: false,
+                width: 360,
+                height: 80
+            });
+        };
+
+        const clearDownloadedEpisodes = function(rules) {
+            new MochaUI.Window({
+                id: 'clearRulesPage',
+                title: 'QBT_TR(New rule name)QBT_TR[CONTEXT=AutomatedRssDownloader]',
+                loadMethod: 'iframe',
+                contentURL: 'confirmruleclear.html?rules=' + encodeURIComponent(rules.join('|')),
+                scrollbars: false,
+                resizable: false,
+                maximizable: false,
+                width: 350,
+                height: 85
+            });
+        }
+
+        const saveSettings = function() {
+            let lastSelectedRow = rssDownloaderRulesTable.selectedRows[rssDownloaderRulesTable.selectedRows.length - 1];
+            let rule = rssDownloaderRulesTable.rows[lastSelectedRow].full_data.name;
+
+            rulesList[rule].useRegex = $('useRegEx').checked;
+            rulesList[rule].mustContain = $('mustContain_text').value;
+            rulesList[rule].mustNotContain = $('mustNotContain_text').value;
+            rulesList[rule].episodeFilter = $('episodeFilter_text').value;
+            rulesList[rule].smartFilter = $('useSmartFilter').checked;
+            rulesList[rule].assignedCategory = $('assignCategory_combobox').value;
+            rulesList[rule].savePath = $('savetoDifferentDir').checked ? $('saveTo_text').value : "";
+            rulesList[rule].ignoreDays = parseInt($('ignoreDaysValue').value);
+
+            switch ($('addPaused_combobox').value) {
+                case "default":
+                    rulesList[rule].addPaused = null;
+                    break;
+                case "always":
+                    rulesList[rule].addPaused = true;
+                    break;
+                case "never":
+                    rulesList[rule].addPaused = false;
```

Comment:
Please provide a `break` statement in the last case. It reduces the likelihood that someone will forget to add it, should they ever add additional cases.